### ISSUE TITLE
Feature Request: Allow to increase/reduce font size

### ISSUE THREAD

**Issue by piotryordanov** [keybase/keybase-issues#2988]
Hello!

I'm using the latest keybase mac app. Is it possible to increase or reduce the font size?

Cheers!

**Comment by Ashaman-** [comment]
Did you ever discover a workaround for this problem?

**Comment by landry314** [comment]
Yes, the font is too small! It makes the program almost unusable. I would love to change the font, size, and style... at least the size.

**Comment by oconnor663** [comment]
What platforms are you all on? If Linux, is there any chance the font gets bigger if you run `run_keybase` manually?

**Comment by Ashaman-** [comment]
@landry314 and I are both on OSX. I'm on High Sierra, not sure about him, although it's less of a problem for me. I have no issue on my Linux machine.
